addons/website_mail_channel/controllers/main.py

```diff
@@ -52,12 +42,11 @@ def view(self, **post):
     @http.route(["/groups/subscription/"], type='json', auth="user")
     def subscription(self, group_id=0, action=False, **post):
         """ TDE FIXME: seems dead code """
-        cr, uid, context = request.cr, request.uid, request.context
-        group_obj = request.registry.get('mail.channel')
+        Group = request.env['mail.channel']
         if action:
-            group_obj.message_subscribe_users(cr, uid, [group_id], context=context)
+            Group.message_subscribe_users(user_ids=[group_id])
```

Comment:
Call functions message_subscribe_users/message_unsubscribe_users  with "mail.channel" recordset of "group_id".